static/static/csv/GeoNames-Monthly-Diff-Report-October-2021.csv

```diff
@@ -0,0 +1,156 @@
+city_name,9935902,9935902,Chuanshan,Chuanshancun,China,China,CN,CN,GX,GX
+city_name,1811200,1811200,Fengxian,Fengcheng,China,China,CN,CN,JS,JS
+"city_name, geoname_id",1788694,10859300,Xinpu,Lianyungang,China,China,CN,CN,JS,JS
+"city_name, geoname_id",6138802,11257133,Saint-Michel-des-Saints,Saint-Alexis-des-Monts,Canada,Canada,CA,CA,QC,QC
+"city_name, geoname_id",3385660,3402320,Urucuba,Cedro,Brazil,Brazil,BR,BR,PE,PE
+city_name,9903037,9903037,Hongqi,Hongqicun,China,China,CN,CN,GX,GX
+city_name,9996923,9996923,Baosha,Baoshacun,China,China,CN,CN,GX,GX
+city_name,1787816,1787816,Yucheng,Ya'an,China,China,CN,CN,SC,SC
+"city_name, geoname_id",1174872,11246240,Karachi,Karachi Water and Severage Colony,Pakistan,Pakistan,PK,PK,SD,SD
+"city_name, geoname_id",1167528,1185021,Quetta,Kili Rahmanzai,Pakistan,Pakistan,PK,PK,BA,BA
+city_name,1909945,1909945,Baiyu,Baiyucun,China,China,CN,CN,FJ,FJ
+"city_name, geoname_id",2847251,2847213,Rieden,Rieder,Germany,Germany,DE,DE,BY,BY
+city_name,1815482,1815482,Kuiju,Changyi,China,China,CN,CN,SD,SD
+city_name,8359315,8359315,Luhe,Luehe,China,China,CN,CN,JS,JS
+city_name,9895479,9895479,Hechi,Hechicun,China,China,CN,CN,GX,GX
+city_name,9929034,9929034,Yi'an,Yi'ancun,China,China,CN,CN,GX,GX
+city_name,8429009,8429009,Fengjia,Fengjiacun,China,China,CN,CN,SD,SD
+"city_name, geoname_id",5924579,6092122,Cobourg,Oakville,Canada,Canada,CA,CA,ON,ON
+"city_name, geoname_id, subdivision_codes",2760043,2751810,Achterbroek,Leerbroek,Netherlands,Netherlands,NL,NL,ZH,UT
+"city_name, geoname_id",3681008,3681010,Municipio Hato Corozal,Hato Corozal,Colombia,Colombia,CO,CO,CAS,CAS
+city_name,1817421,1817421,Baishan,Baishancun,China,China,CN,CN,GX,GX
+city_name,1909600,1909600,Dahe,Dahecun,China,China,CN,CN,GX,GX
+"city_name, geoname_id, subdivision_codes",1415765,7091688,Ukhi,Oghi,Pakistan,Pakistan,PK,PK,JK,KP
+city_name,9929439,9929439,Shifeng,Shifengcun,China,China,CN,CN,GX,GX
+city_name,1817582,1817582,Bailong,Bailong Shequ,China,China,CN,CN,GX,GX
+"city_name, geoname_id",1162812,1162813,Chak Sixty-one -Twelve Left,Vehari,Pakistan,Pakistan,PK,PK,PB,PB
+city_name,3623715,3623715,Fortuna,La Fortuna,Costa Rica,Costa Rica,CR,CR,G,G
+"city_name, geoname_id",6355181,12319446,Les Laures,Les Maurels,France,France,FR,FR,"PAC, 83","PAC, 83"
+"city_name, geoname_id",8299623,2640923,Orkney,Orkney Islands,United Kingdom,United Kingdom,GB,GB,"SCT, ORK","SCT, ORK"
+"geoname_id, subdivision_codes",7522443,862995,,,Estonia,Estonia,EE,EE,"37, 141","37, 784"
+city_name,2836443,2836443,Schoenwalde,Schoenwalde-Glien,Germany,Germany,DE,DE,BB,BB
+city_name,2836443,2836443,Schoenwalde,Schoenwalde-Glien,Germany,Germany,DE,DE,BB,BB
+city_name,704737,704737,Kostopil',Kostopil,Ukraine,Ukraine,UA,UA,56,56
+"city_name, geoname_id",1482597,1170951,Pansera,Mansehra,Pakistan,Pakistan,PK,PK,KP,KP
+country_name,1645457,1645457,Dili,Dili,East Timor,Democratic Republic of Timor-Leste,TL,TL,DI,DI

```

Comment:
I mentioned this in Slack, but these are `Timor-Leste` now, not `Democratic Republic of Timor-Leste`.